src/cc_catalog_airflow/dags/provider_api_scripts/finnish_museums.py

```diff
@@ -137,9 +140,9 @@ def _get_raw_tags(obj):
 
 def _get_landing(obj, landing_url=LANDING_URL):
     l_url = None
-    id = obj.get("id")
-    if id:
-        l_url = landing_url + id
+    id_ = obj.get("id")
+    if id_:
+        l_url = landing_url + id_
```

Comment:
Both [`id`](https://docs.python.org/3/library/functions.html#id) and [`license` ](https://docs.python.org/3/library/constants.html#license)are reserved words in Python: id is a built-in function, and license is a built-in constant. So it's a good practice not to use them as variable names. 

PEP8 suggests that to avoid name collision when you want to use the built in name for your variable, you can use a single trailing underscore (`id` -> `id_`):

https://pep8.org/#descriptive-naming-styles
> single_trailing_underscore_: used by convention to avoid conflicts with Python keyword, e.g.:
> 
> Tkinter.Toplevel(master, class_='ClassName')

Thank you for asking, I learned a bit more about it while researching the answer :)